README.md

```diff
@@ -1,6 +1,25 @@
-# rolemodel-design-system
+# Rolemodel Design System
 
-An experiment in breaking the styles and design system elements out of RoleModel Rails and also experiment with packages.
+RoleModel Design System is a scss package that provides base styles and components that can be integrated and customized in a variety of projects.
+
+## Installation
+
+### NPM
+```sh
+npm install @rolemodel/rolemodel-design-system
+```
+
+### Yarn
+```sh
+yarn add @rolemodel/rolemodel-design-system
+```
+
+### Import
+
+To use the package, add this import to the top of your root level `scss` file.
+```scss
+@import '@rolemodel/rolemodel-design-system';
```

Comment:
If using Webpack to compile your CSS. If using dart-sass with PostCSS, you might need to import it like this:
```
@import '@rolemodel/rolemodel-design-system/dist/scss/rolemodel-design-system'
```